### PR TITLE
BUG: fix pandas compat

### LINKED ISSUE #5891: Remove Panel from pandas compat
pandas is about to remove Panel altogether, in which case we'll see errors in compat.pandas.

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [x] closes #5891

pandas is about to release 0.25.0, which will finally remove Panel.

### CODE REVIEW

File: statsmodels/compat/pandas.py [hunk 1]
```diff
@@ -8,14 +8,18 @@
...
+pandas_ge_25_0 = version >= LooseVersion('0.25.0')
 
 try:
     from pandas.api.types import is_numeric_dtype  # noqa:F401
 except ImportError:
     from pandas.core.common import is_numeric_dtype  # noqa:F401
 
-if version >= '0.20':
+if pandas_ge_25_0:
+    from pandas.tseries import frequencies  # noqa:
```
Comment (bashtage) [inline statsmodels/compat/pandas.py]: what about frequencies?

Comment (jbrockmendel) [reply statsmodels/compat/pandas.py]: fixed typo for the end of this line.  Were you referring to something else?

How much of this do we actually use?  e.g. pandas.util.testing we frequently import directly in the tests.

Comment (bashtage) [reply statsmodels/compat/pandas.py]: I missed frequencies since the try/ex wasn't needed.   LGTM.

Comment (jbrockmendel) [reply statsmodels/compat/pandas.py]: Great.  If we can get this into a 0.10.1 we can prevent a months-long repeat of the scipy import errors.

**Comment by ChadFulton** [comment]
So from this it seems that we will want to plan to release a v0.10.1 fairly soon?

**Comment by bashtage** [comment]
Merging despite the fail.  Looks right and strange CI issues that are unrelated.

**Pull request merged**